src/03-modality-agnostic-files.md

```diff
@@ -379,6 +381,34 @@ meg/sub-control01_task-rest_split-01_meg.nii.gz	1877-06-15T12:15:27
 meg/sub-control01_task-rest_split-02_meg.nii.gz	1877-06-15T12:15:27
 ```
 
+## Sessions file
+
+Template:
+
+```Text
+sub-<label>/
+            ses-<label>/
+            sub-<label>_sessions.tsv
```

Comment:
the indentations for these "trees" seem a bit inconsistent across this PR - but that's probably an issue across the whole spec. I think I would do "4 spaces" per indentation level.